tests/integration/test_fast_color_refine.py

```diff
@@ -57,6 +59,5 @@ def test_files(self):
                 self.assertEqual(result[0], result[1], result[2])
                 print(result[2], 'got', result[1])
```

Comment:
Ik denk dat we deze beter kunnen vervangen door 'debug(...)' zodat dit alleen print als DEBUG = True, misschien kunnen we wil de file/graph name printen. Anders print de test heel veel dingen.